Library/Homebrew/caveats.rb

```diff
@@ -56,6 +56,9 @@ def keg_only_text(skip_reason: false)
       <<~EOS
         #{f.name} is keg-only, which means it was not symlinked into #{HOMEBREW_PREFIX},
         because #{f.keg_only_reason.to_s.chomp}.
+        If #{f.name} is a dependency library for another package you are
+        installing, the correct PATH's were used internally and no modification is needed.
+        See also https://github.com/Homebrew/brew/blob/master/docs/FAQ.md#what-does-keg-only-mean
```

Comment:
```suggestion

        See #{Formatter.url("https://docs.brew.sh/FAQ#what-does-keg-only-mean")}.
```
Let's keep the messaging to the absolute minimum. Many/most users don't read them, and making them wordier just increases the number of users who don't read the messages.

You'll need to adjust the test appropriately.